### ISSUE TITLE
[Create private room] Picture doesn't not displayed

### ISSUE THREAD

**Issue by Claire1817** [tchapgouv/tchap-android#394]
The pictures set during the room creation is not displayed when the room has been created.
When we clicked on the avatar in the room settings a black screen appears (there is an url ? Antivirus problem ?

**Comment by giomfo** [comment]
this is not systematic, to be investigated

**Comment by Claire1817** [comment]
Prod: i didn't reproduce the bug 
Preprod: I reproduce the bug. 
Stacktrace:
```
2022-03-02 11:30:09.270 26557-27966/fr.gouv.rie.tchap V/FormattedJsonHttpLogger: <-- 404 Not Found https://matrix.i.tchap.gouv.fr/_matrix/media_proxy/unstable/download/i.tchap.gouv.fr/rhvRXLdISxZVQRcvSCwyFZQe (26ms, 167-byte body)
2022-03-02 11:30:09.271 26557-26807/fr.gouv.rie.tchap E/GlideExecutor: Request threw uncaught throwable
    com.bumptech.glide.Registry$NoSourceEncoderAvailableException: Failed to find source encoder for data class: class android.net.Uri$StringUri
        at com.bumptech.glide.Registry.getSourceEncoder(Registry.java:578)
        at com.bumptech.glide.load.engine.DecodeHelper.getSourceEncoder(DecodeHelper.java:242)
        at com.bumptech.glide.load.engine.SourceGenerator.cacheData(SourceGenerator.java:104)
        at com.bumptech.glide.load.engine.SourceGenerator.startNext(SourceGenerator.java:47)
        at com.bumptech.glide.load.engine.DecodeJob.runGenerators(DecodeJob.java:311)
        at com.bumptech.glide.load.engine.DecodeJob.runWrapped(DecodeJob.java:280)
        at com.bumptech.glide.load.engine.DecodeJob.run(DecodeJob.java:235)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
        at com.bumptech.glide.load.engine.executor.GlideExecutor$DefaultThreadFactory$1.run(GlideExecutor.java:393)
 ```

**Comment by Claire1817** [comment]
Bug come from Element:
https://github.com/vector-im/element-android/issues/5402

**Comment by giomfo** [comment]
would be fixed after rebase #497
